mithril-signer/src/configuration.rs

```diff
@@ -121,6 +121,15 @@ pub struct Configuration {
 
     /// Preloading refresh interval in seconds
     pub preloading_refresh_interval_in_seconds: u64,
+
+    /// Number of retry attempts when publishing the signature
+    pub signature_publisher_retry_attempts: usize,
```

Comment:
Maybe this is enough?
```suggestion
    pub signature_publisher_retry_attempts: u8,
```

Reply:
Agree, I made the change.